js/exporter/jspdf/pdf_table.js

```diff
@@ -2,16 +2,35 @@ import { isDefined } from '../../core/utils/type';
 import { drawPdfTable } from './draw_pdf_table';
 
 export class PdfTable {
-    constructor(drawTableBorder, rect) {
+    constructor(drawTableBorder, rect, columnWidths) {
         this.drawTableBorder = drawTableBorder;
         this.rect = rect;
+        this.columnWidths = columnWidths ?? [];
         this.rows = [];
     }
 
-    addRow(cells) {
+    tryGetXPos(cellIndex) {
+        return this.rect.x + this.columnWidths.slice(0, cellIndex).reduce((a, b) => a + b, 0);
+    }
+
+    tryGetYPos(cellIndex) {
+        let currentYPos = this.rect.y;
+        for(let index = 0; index < this.rows.length - 1; index++) {
+            const cell = this.rows[index][cellIndex];
+            if(cell && cell.rect.h) {
```

Comment:
There is the 'if(!cells[i].skip)' condition and 'rect' will not be initialized.
How code will manage 'height' in this case?

Reply:
Cells with cells[i].skip === true are not participate in calculation. Their sizes were merged with master cell ( Cells with attributes: colSpan, rowSpan )